contracts/Everlong.sol

```diff
@@ -1,22 +1,23 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity 0.8.22;
+pragma solidity ^0.8.20;
```

Comment:
What's the rationale for this change? IMO it's good to lock the version for the core contracts and keep the version unlocked for libraries and interfaces that will be used elsewhere.

Reply:
Unintentional. Leftover from debugging a compiler version issue (ended up being from an oz contract upstream import). Will update with pinned version